apps/backend/.env.example

```diff
@@ -22,7 +22,6 @@ DB_POOL_MAX=10
 # JWT
 #
 JWT_SECRET=<SECRET_PHRASE>
-JWT_EXPIRATION_TIME=24hr
```

Comment:
forgot to move jwt expiration variables into env

Reply:
Not fixed